cosmos/nyx.json

```diff
@@ -0,0 +1,48 @@
+{
+  "chainId": "nyx",
+  "chainName": "Nyx",
+  "chainSymbolImageUrl": "https://github.com/nymtech/keplr-chain-registry/blob/main/images/nyx/chain.png",
+  "rpc": "https://rpc.nyx.nodes.guru",
+  "rest": "https://api.nyx.nodes.guru/",
+  "bip44": { 
+    "coinType": 118 
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "n",
+    "bech32PrefixAccPub": "npub",
+    "bech32PrefixValAddr": "nvaloper",
+    "bech32PrefixValPub": "nvaloperpub",
+    "bech32PrefixConsAddr": "nvalcons",
+    "bech32PrefixConsPub": "nvalconspub"
+  },
+  "stakeCurrency": {
+    "coinDenom": "nyx",
+    "coinMinimalDenom": "unyx",
+    "coinDecimals": 6
```

Comment:
Is it right coin to stake?

Reply:
Yes, this is the coin that will be staked on the blockchain - `NYM` token is only used for the mixnet.